docs/en-us/doc_flask.md

```diff
@@ -104,9 +106,108 @@ Here's what you will get:
 ![flask-0](https://github.com/chenjiandongx/pyecharts/blob/master/images/flask-0.gif)
 
 
-## Conclusion
+### Conclusion
 
 It is extremely simple to include a pyexcharts instance in a Flask app. Please go
 ahead and put your own charts in flask.
 
 For your reference, please find the example code in `pyecharts/document` folder.
+
+
+
+## Way II :Render Using Template Functions
+
+From V0.3.0, pyecharts is support to use `echarts_*`  template functions in Flask web framework.
```

Comment:
pyecharts exposes `echarts_*`  template functions which can be directly used within Flask web framework.